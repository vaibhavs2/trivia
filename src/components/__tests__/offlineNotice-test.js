/**
 * @flow strict-local
 */
import React from 'react';
import renderer from 'react-test-renderer';

import {OfflineNotice} from '../';

test('Offline-Notice renders correctly', () => {
  const tree = renderer.create(<OfflineNotice />).toJSON();
  expect(tree).toMatchSnapshot();
});
