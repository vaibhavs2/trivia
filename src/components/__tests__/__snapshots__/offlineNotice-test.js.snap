// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Offline-Notice renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "red",
      }
    }
  >
    You don't have Internet connection 
  </Text>
</View>
`;
